Source/Core/Core/State.cpp

```diff
@@ -74,7 +74,7 @@ static Common::Event g_compressAndDumpStateSyncEvent;
 static std::thread g_save_thread;
 
 // Don't forget to increase this after doing changes on the savestate system
-constexpr u32 STATE_VERSION = 129;  // Last changed in PR 9511
+constexpr u32 STATE_VERSION = 130;  // Last changed in PR 953x
```

Comment:
Close enough, this is 9545 ;)

Reply:
Whoops, good catch... Fixed in the last commit now